app/src/main/java/org/mozilla/rocket/shopping/search/ui/ShoppingSearchKeywordInputViewModel.kt

```diff
@@ -34,7 +34,7 @@ class ShoppingSearchKeywordInputViewModel(
 
     init {
         isFirstRun = shouldShowSearchInputOnboarding()
-        emitUiModel(ShoppingSearchKeywordInputUiModel(hideClear = true, hideHintContainer = !isFirstRun))
+        emitUiModel(ShoppingSearchKeywordInputUiModel(hideClear = true, hideHintContainer = false))
```

Comment:
@hartsu please help to remove `isFirstRun` and its related code in use case, repository, and data source, thanks!